docs/dev/table/connectors/kafka.md

```diff
@@ -0,0 +1,175 @@
+See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/projectsetup/dependencies.html).
+
+How to create a Kafka table
+----------------
+<div class="codetabs" markdown="1">
+<div data-lang="SQL" markdown="1">
+{% highlight sql %}
+CREATE TABLE kafkaTable (
+ ...
+) WITH (
+ 'connector' = 'kafka',
+ 'topic' = 'topic_name',
+ 'properties.bootstrap.server' = 'localhost:9092',
+ 'properties.group.id' = 'testGroup',
+ 'format' = 'csv',
+ 'scan.startup.mode' = 'group-offsets'
+)
+{% endhighlight %}
+</div>
+</div>
+
+Connector Options
+----------------
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>connector</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what connector to use, for Kafka the options are: 'kafka', 'kafka-0.11', 'kafka-0.10'.</td>
+    </tr>
+    <tr>
+      <td><h5>topic</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required topic name from which the table is read.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.bootstrap.servers</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required Kafka server connection string.</td>
+    </tr>
+    <tr>
+    <td><h5>format</h5></td>
+    <td>required</td>
+    <td style="word-wrap: break-word;">(none)</td>
+    <td>String</td>
+    <td>Kafka connector requires to specify a format,
+    the supported formats are 'csv', 'json' and 'avro'.
+    Please refer to [Table Formats]({{ site.baseurl }}/dev/table/connect.html#table-formats) section for more details.
+    </td>
+        </tr>
+    <tr>
+      <td><h5>properties.group.id</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required consumer group in Kafka consumer, no need for Kafka producer</td>
+    </tr>
+    <tr>
+      <td><h5>scan.startup.mode</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">group-offsets</td>
+      <td>String</td>
+      <td>Optional startup mode for Kafka consumer, valid enumerations are 'earliest-offset', 'latest-offset', 'group-offsets', 'timestamp' or 'specific-offsets'.</td>
+    </tr>
+    <tr>
+      <td><h5>scan.startup.specific-offsets</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Optional offsets used in case of 'specific-offsets' startup mode.
```

Comment:
Specifies offsets for each partition in case of 'specific-offsets' startup mode, e.g. `partition:0,offset:42;partition:1,offset:300`.